PowerGit.csproj

```
@@ -1,11 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibGit2Sharp" Version="0.26.1" />
-  </ItemGroup>
+    <PackageReference Include="LibGit2Sharp" Version="0.26.1" />
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.289" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="$(PkgLibGit2Sharp_NativeBinaries)\**\runtimes\**\*" CopyToOutputDirectory="PreserveNewest" />

```

Shouldn't this be the specific version instead of matching all versions?
```suggestion
    <Content Include="$(PkgLibGit2Sharp_NativeBinaries)\2.0.289\runtimes\**\*" CopyToOutputDirectory="PreserveNewest" />
```

---

This change actually fixes the perf problem

---

The problem with this change is that you no longer get a `runtimes` folder in your output. Instead you end up with all of the folders inside the `runtimes` folder in the output.

---

Yeah, I noticed that and tried to fix it with `Link` or `LinkBase`, but it didn't work. It's really not an issue though, the subfolder doesn't matter, I need to move the right runtime file anyway at first run for libgit2sharp to pick it up. But I definitely need to be able to build this on a mac :)